presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/DecimalConfig.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.math.RoundingMode;
+
+import static java.math.RoundingMode.UNNECESSARY;
+
+public class DecimalConfig
+{
+    private DecimalMapping decimalMapping = DecimalMapping.STRICT;
+    private int decimalDefaultScale = 16;
```

Comment:
What if we we make it to default to 0 to "force" users to choose their own default?

Reply:
That's a good option since the default rounding mode is `UNNECESSARY` and by switching to `allow_overflow` select statements will fail with `Rounding necessary` error for any value that is not integer.
I will change the default to `0`